src/js/clipboard.js

```diff
@@ -0,0 +1,640 @@
+/*!
+ * clipboard.js v2.0.0
+ * https://zenorocha.github.io/clipboard.js
+ *
+ * Licensed MIT © Zeno Rocha
+ */
+!(function (t, e) {
+  "object" == typeof exports && "object" == typeof module
+    ? (module.exports = e())
+    : "function" == typeof define && define.amd
+    ? define([], e)
+    : "object" == typeof exports
+    ? (exports.ClipboardJS = e())
+    : (t.ClipboardJS = e());
```

Comment:
Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)